src/components/ScheduleFieldset.vue

```diff
@@ -33,12 +32,12 @@
 <script lang="ts" setup>
   import { computed } from 'vue'
   import ScheduleFormModal from '@/components/ScheduleFormModal.vue'
-  import { useCan } from '@/compositions/useCan'
   import { Schedule } from '@/models'
 
   const props = defineProps<{
     modelValue: Schedule | null,
     loading?: boolean,
+    readonly?: boolean,
```

Comment:
This seems a little risky since it defaults to `false`. When really we need the implementation to tell us this. This component could easily be implemented without this being set and show a user an action they don't have permission to complete. 

The simplest way might be to just make `readonly` a required prop.

Reply:
👍 - I don't see any outside consumption of this component in oss or cloud so making it required should be non-breaking update for consumers.